### DISCUSSION TITLE
Link de entrega de ejercicio en VUE

### DISCUSSION THREAD

**Discussion by sergiolopezloya** [LaunchX-InnovaccionVirtual/FrontEnd-Mision#562] [Q&A]
Hola compañeros, apenas ayer tuve oportunidad de terminar mi práctica de VUE (requirió más tiempo que otras prácticas). Alguien sabe donde está el link para mandarla? Que no encuentro por ningún lado en Teams

**Comment by MarcosSilvaG** [comment]
Creo que aún no se ha publicado

**Comment by doguedogue** [comment]
Mañana dirán al respecto, no se publicó forma como las otras prácticas.
